src/main/java/uk/gov/hmcts/sptribs/caseworker/event/CaseworkerDocumentManagementRemove.java

```diff
@@ -166,6 +164,18 @@ private CicCase removeOrderDoc(CicCase cicCase) {
         return cicCase;
     }
 
+    private void manageUploadedFiles(ListValue<Order> orderListValue, ListValue<CaseworkerCICDocument> cicDocumentListValue) {
+        if (!CollectionUtils.isEmpty(orderListValue.getValue().getUploadedFile())) {
+            for (int i = 0; i < orderListValue.getValue().getUploadedFile().size(); i++) {
+                ListValue<CICDocument> file = orderListValue.getValue().getUploadedFile().get(i);
+                if (null != file.getValue().getDocumentLink() && file.getValue().getDocumentLink()
```

Comment:
Just a suggestion: could we use Objects.equals() here, as it's null safe?

https://docs.oracle.com/javase/7/docs/api/java/util/Objects.html#equals(java.lang.Object,%20java.lang.Object)

Probably does the same under the covers but might be slightly more readable and concise...

Reply:
Does not work same actually, Object might change because we create another list, so I check on document link